test/NosCore.Tests/InventoryTests/ShopTests.cs

```diff
@@ -259,15 +259,15 @@ public async Task UserCanShopReput()
                 InventoryItemInstance.Create(itemBuilder.Create(1, 1), _session.Character.CharacterId),
                 NoscorePocketType.Etc, 2);
 
-            await _session.Character.Buy(shop, 0, 998).ConfigureAwait(false);
+            await _session.Character.BuyAsync(shop, 0, 998).ConfigureAwait(false);
             Assert.IsTrue(_session.Character.InventoryService.All(s => s.Value.ItemInstance?.Amount == 999));
             Assert.IsTrue(_session.Character.Reput == 499002);
         }
 
         private ClientSession PrepareSessionShop()
         {
             var conf = new WorldConfiguration {BackpackSize = 3, MaxItemAmount = 999, MaxGoldAmount = 999_999_999};
-            var session2 = new ClientSession(conf, Logger, new List<IPacketHandler>(), _friendHttpClient!, new Mock<ISerializer>().Object, new Mock<IPacketHttpClient>().Object);
+            var session2 = new ClientSession(conf, new Mock<IMapInstanceProvider>().Object, new Mock<IExchangeProvider>().Object, Logger, new List<IPacketHandler>(), _friendHttpClient!, new Mock<ISerializer>().Object, new Mock<IPacketHttpClient>().Object, new Mock<IMinilandProvider>().Object);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 294 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5266673)